ui/src/app/views/workspace-wrapper/component.ts

```diff
@@ -14,29 +15,29 @@ import {
 } from 'generated';
 
 @Component({
-  selector: 'app-workspace-nav-bar',
   styleUrls: ['../../styles/buttons.css',
-    '../../styles/headers.css',
-    './component.css'],
+    '../../styles/headers.css'],
   templateUrl: './component.html',
 })
-export class WorkspaceNavBarComponent implements OnInit, OnDestroy {
+export class WorkspaceWrapperComponent implements OnInit, OnDestroy {
   @ViewChild(WorkspaceShareComponent)
   shareModal: WorkspaceShareComponent;
 
   workspace: Workspace;
-  wsId: string;
-  wsNamespace: string;
   accessLevel: WorkspaceAccessLevel;
   deleting = false;
+  sharing = false;
   workspaceDeletionError = false;
   tabPath: string;
-  display = true;
+  displayNavBar = true;
   confirmDeleting = false;
 
   @ViewChild(BugReportComponent)
   bugReportComponent: BugReportComponent;
 
+  @ViewChild(WorkspaceNavBarComponent)
+  workspaceNavBarComponent: WorkspaceNavBarComponent;
```

Comment:
This shouldn't be necessary since you don't utilize the reference anywhere. `@ViewChild` is only needed if you have to call a method or something on a child component.